src/main/java/net/sf/jabref/importer/fetcher/ACMPortalFetcher.java

```diff
@@ -452,6 +449,7 @@ private String getResults(URL source) throws IOException {
                 sb.append((char) buffer[i]);
             }
         }
+        in.close();
```

Comment:
use try-with-resources instead. This is the idiom to ensure that any opened resource is closed for sure. 

```java
try(InputStream in = source.openStream()) {
   // use in
}
```